tools/src/commands/ValidateWorkspaceDependencies.ts

```diff
@@ -0,0 +1,75 @@
+import { Command } from '@expo/commander';
+import path from 'path';
+
+import { getExpoRepositoryRootDir } from '../Directories';
+import { getInfoAsync } from '../Workspace';
+
+type ActionOptions = {
+  workspace?: string;
+};
+
+export default (program: Command) => {
+  program
+    .command('validate-workspace-dependencies')
+    .alias('vwd')
+    .description('Verifies if all workspaces are linked from monorepo.')
```

Comment:
A workspace refers to the set of packages. I've been using the term workspace package to refer to a single package in a workspace:

```suggestion
    .description('Verifies if all workspace packages are linked from monorepo.')
```

This feedback applies throughout this PR, I didn't leave more comments about it.

Reply:
Changed them all. Will "request re-review" to get rid of the PR blocker.